scripts/test/install-vpc-cni.sh

```diff
@@ -5,7 +5,7 @@
 set -eo pipefail
 
 SCRIPTS_DIR=$(cd "$(dirname "$0")" || exit 1; pwd)
-DEFAULT_VPC_CNI_VERSION="v1.7.10"
+DEFAULT_VPC_CNI_VERSION="1.11"
```

Comment:
Did you mean 1.11.0? we can use the latest 1.11.3.

Reply:
I wanted to pin the VPC CNI version to v1.11.0 for testing since that is the recommended version.

Reply:
I wanted to pin the VPC CNI version to v1.11.0 for testing since that is the recommended version.

Reply:
I've resolved this. We will use the latest patch version available for the release for testing as well.